src/Models/Address.php

```diff
@@ -0,0 +1,161 @@
+<?php namespace DivideBV\PHPDivideIQ\Models;
+
+/**
+ * Created by PhpStorm.
+ * User: Nav
+ * Date: 28-07-16
+ * Time: 12:14
+ */
+class Address
+{
+    /**
+     * required
+     * @var string
+     */
+    protected $Street;
+    /**
+     * required
+     * @var string
+     */
+    protected $StreetNumber;
+    /**
+     * @var string
+     */
+    protected $StreetNumberAdditition;
+    /**
+     * required
+     * @var string
+     */
+    protected $ZipCode;
+    /**
+     * required
+     * @var string
+     */
+    protected $City;
+    /**
+     * required
+     * CountryCode (ISO2 or ISO3).
+     * @var string
+     */
+    protected $CountryCode;
+
+    /**
+     * @return string
+     */
+    public function getStreet()
+    {
+        return $this->Street;
+    }
+
+    /**
+     * @param string $Street
+     * @return Address
+     */
+    public function setStreet($Street)
+    {
+        $this->Street = $Street;
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getStreetNumber()
+    {
+        return $this->StreetNumber;
+    }
+
+    /**
+     * @param string $StreetNumber
+     * @return Address
+     */
+    public function setStreetNumber($StreetNumber)
+    {
+        $this->StreetNumber = $StreetNumber;
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getStreetNumberAdditition()
+    {
+        return $this->StreetNumberAdditition;
+    }
+
+    /**
+     * @param string $StreetNumberAdditition
+     * @return Address
+     */
+    public function setStreetNumberAdditition($StreetNumberAdditition)
+    {
+        $this->StreetNumberAdditition = $StreetNumberAdditition;
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getZipcode()
+    {
+        return $this->ZipCode;
+    }
+
+    /**
+     * @param string $ZipCode
+     * @return Address
+     */
+    public function setZipcode($ZipCode)
+    {
+        $this->ZipCode = $ZipCode;
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getCity()
+    {
+        return $this->City;
+    }
+
+    /**
+     * @param string $City
+     * @return Address
+     */
+    public function setCity($City)
+    {
+        $this->City = $City;
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getCountryCode()
+    {
+        return $this->CountryCode;
+    }
+
+    /**
+     * @param string $CountryCode
+     * @return Address
+     */
+    public function setCountryCode($CountryCode)
+    {
+        $this->CountryCode = $CountryCode;
+        return $this;
+    }
+
+    public function toArray()
+    {
+        return array(
```

Comment:
Converted array to short array in commit: https://github.com/DivideBV/PHPDivideIQ/pull/1/commits/19da445b970041225a00f424ad3f2597142fb493